src/main/kotlin/com/cognifide/gradle/aem/common/file/transfer/FileDownloader.kt

```diff
@@ -13,15 +13,20 @@ class FileDownloader(private val aem: AemExtension) {
 
     private var loggedKb: Long = 0
 
-    fun ProgressLogger.logProgress(operation: String, readLength: Long, fullLength: Long, file: File) {
+    fun ProgressLogger.logProgress(operation: String, readLength: Long, fullLength: Long, file: File, nanoStartTime: Long) {
         processedBytes += readLength
 
+        val nanoElapsedTime = System.currentTimeMillis() - nanoStartTime
```

Comment:
I prefer to skip nano prefix as all other is also nano so that such distinguishing / indicating is redundant

Reply:
Did not notice that everything is nano. Renamed the variables. Thanks for the indication.